docs/usage/type-conversions.md

```diff
@@ -1,114 +1,186 @@
-that allow methods and some operators to be called on the object from the other
+that allow methods and some operations to be called on the object from the other
 language.
 
 
-### Javascript from Python
-
-When passing a Javascript object to Python, an extension type is used to
-delegate Python operations to the Javascript side. The following operations are
-currently supported. (More should be possible in the future -- work in ongoing
-to make this more complete):
-
-| Python         | Javascript      |
-|----------------|-----------------|
-| `repr(x)`      | `x.toString()`  |
-| `x.foo`        | `x.foo`         |
-| `x.foo = bar`  | `x.foo = bar`   |
-| `del x.foo`    | `delete x.foo`  |
-| `x(...)`       | `x(...)`        |
-| `x.foo(...)`   | `x.foo(...)`    |
-| `X.new(...)`   | `new X(...)`    |
-| `len(x)`       | `x.length`      |
-| `x[foo]`       | `x[foo]`        |
-| `x[foo] = bar` | `x[foo] = bar`  |
-| `del x[foo]`   | `delete x[foo]` |
-| `x == y`       | `x == y`        |
-| `x.typeof`     | `typeof x`      |
-
-### Python from Javascript
-
-When passing a Python object to Javascript, the Javascript [Proxy
-API](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy)
-is used to delegate Javascript operations to the Python side. In general, the
-Proxy API is more limited than what can be done with a Python extension, so
-there are certain operations that are impossible or more cumbersome when using
-Python from Javascript than vice versa. The most notable limitation is that
-while Python has distinct ways of accessing attributes and items (`x.foo` and
-`x[foo]`), Javascript conflates these two concepts. The following operations are
-currently supported:
-
-| Javascript     | Python                   |
-|----------------|--------------------------|
-| `foo in x`     | `hasattr(x, 'foo')`      |
-| `x.foo`        | `getattr(x, 'foo')`      |
-| `x.foo = bar`  | `setattr(x, 'foo', bar)` |
-| `delete x.foo` | `delattr(x, 'foo')`      |
-| `x.ownKeys()`  | `dir(x)`                 |
-| `x(...)`       | `x(...)`                 |
-| `x.foo(...)`   | `x.foo(...)`             |
-
-An additional limitation is that when passing a Python object to Javascript,
-there is no way for Javascript to automatically garbage collect that object.
-Therefore, custom Python objects must be manually destroyed when passed to Javascript, or
-they will leak. To do this, call `.destroy()` on the object, after which Javascript will no longer have access to the object.
+### Proxying from Javascript into Python
+
+When most Javascript objects are translated into Python a `JsProxy` is returned.
+The following operations are currently supported on a `JsProxy`. (More should be
+possible in the future -- work is ongoing to make this more complete):
+
+| Python                    | Javascript             |
+|---------------------------|------------------------|
+| `repr(proxy)`             | `x.toString()`         |
```

Comment:
More usual would be
```suggestion
| `str(proxy)`             | `x.toString()`         |
```
It would call `__repr__` anyway I think?

Reply:
Confirmed that it calls `__repr__`. (Incidentally, maybe we should try to make `__repr__` return a better quality result for standard objects at some point. `toString` is shockingly low quality.)